test/python/pulse/test_parameters.py

```diff
@@ -179,7 +170,7 @@ def test_substitution(self):
     def test_substitution_with_existing(self):
         """Test that substituting one parameter with an existing parameter works."""
         schedule = pulse.Schedule()
-        schedule += pulse.SetFrequency(self.alpha, DriveChannel(self.qubit))
+        schedule += pulse.SetFrequency(self.alpha, DriveChannel(self.qubit).frame)
```

Comment:
The expectation of changing to ``Channel(i).frame`` should be explicitly documented somewhere.